packs/equipment/beguiling-crown.json

```diff
@@ -37,7 +37,17 @@
             "title": "Pathfinder Treasure Vault"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "definition": [
```

Comment:
```suggestion
                "predicate": [
```

Reply:
I already tested using the predicate of deception and it doesn't work. Never applies the -4 to the DC against deception skill actions. Using definition does though.

Reply:
Try `@Check[type:arcana|defense:perception]` to roll something else against it, it will show the penalty there too. Definition shouldn't work for FlatModifier.

Reply:
Ah, because definition doesn't work, it's functioning like an open FlatModifier. The predicate doesn't work either - I'll see if I can find another workaround.

Reply:
I ended up reverting the changes to the crown as I couldn't figure out how to get it to work.